### ISSUE TITLE
Modal warning dialog about standard PS properties panel

### ISSUE THREAD

**Issue by iwehrman** [adobe-photoshop/spaces-design#3488]
I just launched pgdev.Mac 965, made a new document, and created a rectangle, which apparently triggered this warning:

![image](https://cloud.githubusercontent.com/assets/1075154/12218962/38a92782-b6e5-11e5-917e-420478d1d56d.png)

It now happens every time I create a new shape with a shape tool.

**Comment by iwehrman** [comment]
Over to @chadrolfs to file a JIRA bug.

Any ideas about this @jsbache?

**Comment by iwehrman** [comment]
Still happening for me in pgdev.Mac 967:
![image](https://cloud.githubusercontent.com/assets/1075154/12246343/cf8a9c7a-b862-11e5-886a-6bc8913d6dc1.png)

**Comment by chadrolfs** [comment]
CNR

**Comment by chadrolfs** [comment]
Still want me to file in JIRA?

**Comment by iwehrman** [comment]
I remain suspicious, but since I can't currently repo, I'm closing it. But, I'm itchin' to re-open...

**Comment by iwehrman** [comment]
Just hit this again: 
![image](https://cloud.githubusercontent.com/assets/1075154/12567546/85fcbb8e-c374-11e5-9347-91829018815b.png)

**Comment by iwehrman** [comment]
Tracking issue: https://jira.corp.adobe.com/browse/PS-1073

CC @jsbache

**Comment by iwehrman** [comment]
Reported fixed by @jsbache as of CL 1060779. Repro steps:
```
To consistently repro:
1. Make sure that the properties panel is permanently visible and minimized in standard
  (by expanding the iconic panels)
2. Switch to DS and create a rectangle
```
Over to @chadrolfs to verify.
